.readme-partials.yml

```diff
@@ -1,3 +1,10 @@
 introduction: |-
         [Cloud Pub/Sub](https://cloud.google.com/pubsub/docs) is a fully-managed real-time messaging service that allows
         you to send and receive messages between independent applications.
+
+        This README links to an [API reference](https://googleapis.dev/nodejs/pubsub/latest/index.html#reference), samples,
```

Comment:
So when we go to https://googleapis.dev/nodejs/pubsub/latest/index.html the intro text is going to say *This README* now? If that is the case, it might be better to call it a document or something less specific?

Reply:
maybe:

---

This document contains to an [API reference](https://googleapis.dev/nodejs/pubsub/latest/index.html#reference), samples,
        and other resources useful to developing Node.js applications.
        For additional help developing Pub/Sub applications, in Node.js and other languages, see our
        [Pub/Sub quickstart](https://cloud.google.com/pubsub/docs/quickstart-client-libraries),
        [publisher](https://cloud.google.com/pubsub/docs/publisher), and [subscriber](https://cloud.google.com/pubsub/docs/subscriber)
        guides.